frontend/src/containers/App.js

```diff
@@ -1,11 +1,21 @@
-import React from 'react';
+import React from "react";
 
-import '../styles/App.scss';
+import "../styles/App.scss";
+
+import UniversalLandingPage from "../containers/UniversalLandingPage.js";
+
+import { Route, Switch } from "react-router-dom";
 
 function App() {
   return (
-    <div className='App glow'>
-      
+    <div className="App glow">
+      <Switch>
+        <Route path="/player" />
```

Comment:
You might always want to use prop `exact` in <Route>. 
For example:
```
<Route path="/host" component={Host} />
 <Route path="/host/create" component={CreateGame} />
```

when user access `"/host/create`, <Host> will be rendered, not <CreateGame>